src/lib/requests/getRecordsBySensorId/index.ts

```diff
@@ -1,6 +1,9 @@
 import { supabase } from "@auth/supabase";
 import { definitions } from "@common/types/supabase";
 
+const MAX_DOWNLOADABLE_RECORDS = 1000000;
+const NEXT_PUBLIC_SUPABASE_MAX_ROWS = 5;
```

Comment:
Shouldn't this be defined in an env variable? And set to something other than 5?

Reply:
True! I'm sorry, I think I'm quite tired... Will change!